frontend/Themes.jsx

```diff
@@ -121,26 +125,18 @@ export const DataScrollView = styled.SafeAreaView`
     justify-content: center;
 `;
 
-// Styles for Settings Page:
-export const DetailedSettingsView = styled(PageView)`
-    justify-content: center;
-    align-items: center;
-`;
-
+// Styles for the Settings Page:
```

Comment:
The components under this comment aren't styles for the Settings Page. You could just delete this comment honestly.
```suggestion

```